services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/ResumableFileDownload.java

```diff
@@ -140,6 +281,26 @@ public interface Builder extends CopyableBuilder<Builder, ResumableFileDownload>
          */
         Builder downloadFileRequest(DownloadFileRequest downloadFileRequest);
 
+        /**
+         * The {@link DownloadFileRequest} request
+         *
+         * <p>
+         * This is a convenience method that creates an instance of the {@link DownloadFileRequest} builder avoiding the
+         * need to create one manually via {@link DownloadFileRequest#builder()}.
+         *
+         * @param downloadFileRequestBuilder the download file request builder
+         * @return a reference to this object so that method calls can be chained together.
+         * @see #downloadFileRequest(DownloadFileRequest)
+         */
+        default ResumableFileDownload.Builder downloadFileRequest(Consumer<DownloadFileRequest.Builder>
```

Comment:
Why do we need this? Can't we use `ResumableFileDownload.toBuilder()`?

Reply:
nvm, I got confused

Reply:
I am not sure what you mean with toBuilder? I added the consumer method for more fluent construction of a ResumableFileDownload instance, because it was bugging me that within the DownloadFileRequest, it was possible to add a consumer of GetObjectRequest.